spec/core_functions/math/pow.hrx

```diff
@@ -0,0 +1,320 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#3044
+
+<===>
+================================================================================
+<===> input.scss
+@use "sass:math" as math;
+a {b: math.pow(2, 3)}
```

Comment:
It's probably also a good idea to test a real number with a fractional exponent as well.

Reply:
Done.